src/lang.rs

```diff
@@ -246,6 +248,8 @@ impl ShadowLang {
 
         let mut files = source.files.clone();
         files.sort();
+        let original_path = env::current_dir();
+        let _ = env::set_current_dir(Path::new(&source.dir));
```

Comment:
Minor Nit:
```suggestion
        assert!(env::set_current_dir(Path::new(&source.dir).is_ok());
```
I've seen this used as a fairly common Rust pattern, ie. asserting the Result is `ok` if you don't intend on introspecting the value of the result.

Reply:
This bug came up now for the first time in years. If we can't do the cd for some reason, it's probably fine.

Reply:
👍 Ah ok, fair enough!